backend/migrations/default/1631285997326_alter_table_public_Program_alter_column_ApplicationEnd/up.sql

```diff
@@ -0,0 +1,2 @@
+comment on column "public"."Program"."ApplicationEnd" is E'The default application deadline for a course. It can be changed on the course level thoughthough.';
+alter table "public"."Program" rename column "ApplicationEnd" to "defaultApplicationEndDateTime";
```

Comment:
@ckru-cbb was it on purpose to rename it to `defaultApplicationEnd`? I think the `applicationEnd` is missing in the Program now.

Reply:
@okwast The actual application end is set in the `Course` table. When a new course is added the `applicationEnd `is taken from the `defaultApplicationEnd`. `ApplicationStart `is different since that one is always the same for all courses.

Reply:
Ah, you are right, thank you! 🙇